src/components/Forms/Configs/ConfigListFormField.tsx

```diff
@@ -0,0 +1,66 @@
+import { Field, useFormikContext, FieldArray } from "formik";
+import { FaMinusCircle, FaPlus } from "react-icons/fa";
+import { Button } from "../../Button";
+import { TextInput } from "../../TextInput";
+import { get } from "lodash";
+import { useMemo } from "react";
+
+type ConfigFormFieldProps = {
+  label: string;
+  containerClassName?: string;
+  className?: string;
+  name: string;
+};
+
+export default function ConfigListFormField({
```

Comment:
This seems to be a generic field ?

Reply:
Yeah, that's the idea, it should be used by anyone who needs form fields within formik, that returns a simple array `[1,2,3].